src/components/Buttons/BookMarkButton.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { MdBookmarkBorder, MdBookmark } from "react-icons/md";
+
+function BookMarkButton({
+  isbookMarked = true,
+  isLoading = false,
+  handleBookMark = () => {},
+}) {
+  const handleDisable = (e) => {
```

Comment:
I guess you missed invoking this function, eslint comes in handy in these situations.

Reply:
I have not implement this feature yet.